src/amo/reducers/addonsByAuthors.js

```diff
@@ -172,29 +174,33 @@ const reducer = (
   switch (action.type) {
     case FETCH_ADDONS_BY_AUTHORS: {
       const newState = deepcopy(state);
+      const { addonType, authorUsernames, forAddonSlug } = action.payload;
```

Comment:
Just to be sure: is there anything that has changed in this `case` block? It looks like nothing has been changed besides destructuring `action.payload`.

Reply:
yeh in these 2 cases here it seems that is all that has changed at this point..